.circleci/config.yml

```diff
@@ -208,6 +208,8 @@ jobs:
       - checkout
       - run: go env
       - run:
+          environment: 
+           GOARCH: ""
           name: Install golangci-lint
```

Comment:
After this targeted fix we should get this on main etc.

Reply:
yes